### PR TITLE
[ADD] doc: install on Odoo.sh

### PR DESCRIPTION
**Pull request by Icallhimtest**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/setup/install.rst [hunk 1]
```diff
@@ -564,3 +575,4 @@ default db to serve on localhost:8069
 .. _Editions: https://www.odoo.com/pricing#pricing_table_features
 .. _nightly: https://nightly.odoo.com/10.0/nightly/
 .. _extra: https://nightly.odoo.com/extra/
+.. _Odoo SH: https://www.odoo.sh/
```
Comment (seb-odoo) [inline doc/setup/install.rst]: Do we write `Odoo SH` like that in other places? Shouldn't it be `Odoo.sh`?

Comment (Icallhimtest) [reply doc/setup/install.rst]: dots don't work here :p 
didn't find a way to escape 'em  ^^

Comment (seb-odoo) [reply doc/setup/install.rst]: I couldn't find how to escape it either. Get rid of the variable then and just add the link once with the normal link syntax?

**Comment by Icallhimtest** [comment]
Here you go my friend :)

**Comment by seb-odoo** [comment]
Excellent work!

**Comment by seb-odoo** [comment]
@Icallhimtest I don't have merge rights though

**Comment by Icallhimtest** [comment]
Hello @beledouxdenis,

Could you r+ this? Feel free to drop a comment if you want the paragraph modified.

Thanks!

**Pull request closed without merging**